client/ts/tests/createGlobal.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  Connection,
+  Keypair,
+  sendAndConfirmTransaction,
+  PublicKey,
+  SystemProgram,
+  TransactionInstruction,
+  Transaction,
+} from '@solana/web3.js';
+import { ManifestClient } from '../src/client';
+import { PROGRAM_ID } from '../src/manifest';
+import { Global } from '../src/global';
+import { airdropSol, getClusterFromConnection } from '../src/utils/solana';
+import { createMint } from '@solana/spl-token';
+import { FIXED_MANIFEST_HEADER_SIZE } from '../src/constants';
+
+async function testCreateGlobal(): Promise<void> {
+  const connection: Connection = new Connection('http://127.0.0.1:8899');
+  const payerKeypair: Keypair = Keypair.generate();
+  const globalAddress: PublicKey = await createGlobal(connection, payerKeypair);
+
+  const global: Global = await Global.loadFromAddress({
+    connection,
+    address: globalAddress,
+  });
+  global.prettyPrint();
+}
+
+export async function createGlobal(
+  connection: Connection,
+  payerKeypair: Keypair,
+): Promise<PublicKey> {
+  const globalKeypair: Keypair = Keypair.generate();
```

Comment:
No reason to have a globalKeypair. The global account is at a PDA. Return from this function, if you want it to be the global address, should be
getGlobalAddress(tokenMint)

also, no need to initialize an empty program account here.

Reply:
we call account create inside the global create handler

https://github.com/CKS-Systems/manifest/blob/e9a7e710582aba85042086890aa99acf5760abad/programs/manifest/src/program/processor/global_create.rs#L41